ojd_daps_skills/pipeline/skill_ner_mapping/skill_ner_mapper.py

```diff
@@ -312,7 +319,7 @@ def map_skills(
             logger.error("Trying to map skills using empty dict of skills")
 
         clean_ojo_skill_embeddings = self.bert_model.transform(
-            skill_hashes_filtered.values()
+            list(skill_hashes_filtered.values())
```

Comment:
this one kind of worries me as it would change things a lot I think. I also noticed this variable isn't in the docstring

Reply:
will investigate